NEWS.md

```diff
@@ -1,5 +1,8 @@
 # rsconnect (development version)
 
+* `deployApp()`'s `quarto` argument now takes values `TRUE`, `FALSE` or 
```

Comment:
... Quarto is discovered from `QUARTO_PATH` and `PATH`.

Also: Connect documentation will need updating; we will track this.
https://docs.posit.co/connect/user/publishing-r/#quarto